subprojects/common/report-viewer/src/main/kotlin/com/avito/report/ReportsAddApiImpl.kt

```diff
@@ -200,6 +201,16 @@ internal class ReportsAddApiImpl(private val requestProvider: JsonRpcRequestProv
             reportData["build_id_set"] = mapOf("\$fillSet" to listOf(buildId))
         }
 
+        when (val flakiness = test.flakiness) {
```

Comment:
please add tests (same as for other params)

Reply:
Done